.github/actions/test-ruby/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Run ruby tests
+
+inputs:
+  ruby-version:
+    description: Version of ruby to use with ruby/setup-ruby@v1
+    required: false
+    default: '3.0'
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Set up Ruby
```

Comment:
I'm not sure if you've been able to test this yet, but I think I encountered problems trying to re-use other actions in a composite action. [It looks like maybe that's about to be fixed](https://github.com/actions/runner/pull/612#issuecomment-894551186), but I'm not sure it will work today.

Reply:
Yes, this has been tested
It is actually what is used as part of this PR for the tests

Reply:
Wow, that's great then. It was such a shortcoming that you could not use this before.